draft-ietf-rats-reference-interaction-models.md

```diff
@@ -326,6 +326,74 @@ As soon as the Verifier receives the Evidence and the Event Logs, it appraises t
+     | <-- requestAttestation(handle, authSecIDs, claimSelection) |                                      |
+     |                                                            |                                      |
+  collectClaims(claims, claimSelection)                           |                                      |
+     | => collectedClaims                                         |                                      |
+     |                                                            |                                      |
+  generateEvidence(handle, authSecIDs, collectedClaims)           |                                      |
+     | => evidence                                                |                                      |
+     |                                                            |                                      |
+     | evidence, eventLogs -------------------------------------> |                                      |
+     |                                                            |                                      |
+     |                appraiseEvidence(evidence, eventLogs, refValues)                                   |
+     |                                                            |                                      |
+     |   attestationResults <-----------------------------------  |                                      |
+     |                                                            |                                      |
+     | attestationResults(evidence, results) ----------------------------------------------------------> |      |                                                            |                                      |
+     |                                                            |                                      |      |                                                            |                                      | appraiseResults()
+     |                                                            |                                      |
+~~~~
+
+2. BackGround Check Model
+
+The background-check model is so named because of the resemblance of how employers and volunteer organizations perform background checks. In this model, the attestation sequence is initiated by a Relying Party. The Attestor conveys Evidence to the Relying Party, which does not process its payload, but realys the message and optionally check its signature against a policed trust anchor store. Upon receiving the evidence the Relying Party initiates a session with the Verifier. Once session is established, it forwards the received Evidence to the Verfier. The Verifier, appraises the received Evidence according to its appraisal policy for evidence and returns a coresponding Attestation Result to the Relying Party. The Relying Party then checks the Attestation Result against its own appraisal policy to conclude attestation.
```

Comment:
```suggestion
The background-check model is so named because of the resemblance of how employers and volunteer organizations perform background checks. In this model, the attestation sequence is initiated by a Relying Party. The Attester conveys Evidence to the Relying Party, which does not process its payload, but realys the message and optionally check its signature against a policed trust anchor store. Upon receiving the evidence the Relying Party initiates a session with the Verifier. Once session is established, it forwards the received Evidence to the Verfier. The Verifier, appraises the received Evidence according to its appraisal policy for evidence and returns a corresponding Attestation Result to the Relying Party. The Relying Party then checks the Attestation Result against its own appraisal policy to conclude attestation.
```

Reply:
I think, I have already addressed it!

Thank You!